### PR TITLE
adding metrics for ipvs

### PR DESCRIPTION
**Pull request by mag009**

### CODE REVIEW

**Comment by tas50** [comment]
There's a few things missing here before this can be merged.  You'll want to add a header with requirements and examples similar to the other metrics plugins.  You'll also need a Gemspec for this repo so that Travis can pass tests.

**Comment by mag009** [comment]
yes I saw that I will soon. thanks!

**Comment by eheydrick** [comment]
Bump.

**Comment by mag009** [comment]
done. Completely forgot about this one :) I have a few other plugins to add..

cheers,
MAG

**Comment by eheydrick** [comment]
@mag009 need a rebase here to merge.

**Comment by mag009** [comment]
sorry about that.. here it is. I had a hard time to pass the travis it only fail for 2.2 now but it's not my fault :)

cheers

File: sensu-plugins-ipvs.gemspec [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  s.date                   = Date.today.to_s
+  s.files                  = Dir.glob('{bin,lib}/**/*') + %w(LICENSE README.md CHANGELOG.md)
+  s.executables            = s.files.grep(%r{^bin/}) { |f| File.basename(f) }
+  s.test_files             = s.files.grep(%r{^(test|spec|features)/})
+  s.require_paths          = ["lib"]
+  s.cert_chain             = ["certs/sensu-plugins.pem"]
+  s.signing_key            = File.expand_path(pvt_key) if $PROGRAM_NAME =~ /gem\z/
+  s.platform               = Gem::Platform::RUBY
+  s.required_ruby_version  = '>= 1.9.3'
+
```
Comment (analytically) [inline sensu-plugins-ipvs.gemspec:32]: There should be a runtime dependency on sensu-plugin.

File: sensu-plugins-ipvs.gemspec [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+pvt_key = '~/.ssh/gem-private_key.pem'
+
+Gem::Specification.new do |s|
+  s.name                   = 'sensu-plugins-ipvs'
+  s.version                = SensuPluginsIpvs::VERSION
+  s.authors                = ["Marc-Andre Gatien"]
+  s.email                  = '<m.gatien@gmail.com>'
+  s.homepage               = 'https://github.com/sensu-plugins/sensu-plugins-ipvs'
+  s.summary                = ''
```
Comment (analytically) [inline sensu-plugins-ipvs.gemspec]: Mind adding a summary and description? See other plugins for examples.

File: sensu-plugins-ipvs.gemspec [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  s.license                = 'MIT'
+  s.date                   = Date.today.to_s
+  s.files                  = Dir.glob('{bin,lib}/**/*') + %w(LICENSE README.md CHANGELOG.md)
+  s.executables            = s.files.grep(%r{^bin/}) { |f| File.basename(f) }
+  s.test_files             = s.files.grep(%r{^(test|spec|features)/})
+  s.require_paths          = ["lib"]
+  s.cert_chain             = ["certs/sensu-plugins.pem"]
+  s.signing_key            = File.expand_path(pvt_key) if $PROGRAM_NAME =~ /gem\z/
+  s.platform               = Gem::Platform::RUBY
+  s.required_ruby_version  = '>= 1.9.3'
```
Comment (analytically) [inline sensu-plugins-ipvs.gemspec:31]: Let's start with 2.0. We're dropping 1.9.3 from Feb 2016.

**Comment by mag009** [comment]
Finally !!!

**Pull request merged**